components/script/textinput.rs

```diff
@@ -483,21 +566,37 @@ impl<T: ClipboardProvider> TextInput<T> {
                               mods: KeyModifiers) -> KeyReaction {
         let maybe_select = if mods.contains(SHIFT) { Selection::Selected } else { Selection::NotSelected };
         match (printable, key) {
+            (_, Key::B) if mods.contains(CONTROL | ALT) => {
+                self.adjust_horizontal_by_word(Direction::Backward, maybe_select);
+                KeyReaction::RedrawSelection
+            }
+            (_, Key::F) if mods.contains(CONTROL | ALT) => {
+                self.adjust_horizontal_by_word(Direction::Forward, maybe_select);
+                KeyReaction::RedrawSelection
+            }
+            (_, Key::A) if mods.contains(CONTROL | ALT) => {
+                self.adjust_horizontal_to_end(Direction::Backward, maybe_select);
+                KeyReaction::RedrawSelection
+            }
+            (_, Key::E) if mods.contains(CONTROL | ALT) => {
+                self.adjust_horizontal_to_end(Direction::Forward, maybe_select);
+                KeyReaction::RedrawSelection
+            }
```

Comment:
this will be for if CONTROL + ALT + F are all pressed together, so in theory it should not trigger the CTRL-F shortcut. I can add that as a test though